File: Python/ValidPalindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = ''.join(ch.lower() for ch in s if ch.isalnum())
        for i in range(0,int(len(s)/2)):
            if s[i]!=s[len(s)-i-1]: return False
        return True;

File: LeetCodeProblems/Strings/Palindrome/ValidPalindrome.cpp
using namespace std;

class Solution {
public:
    bool isPalindrome(string s) {
        int pos=0;
        for(int i=0;i<s.size();i++)
        {
            if((s[i]>='a' && s[i]<='z') || (s[i]>='0' && s[i]<='9')) s[pos++] = s[i];
            else if(s[i]>='A' && s[i]<='Z') s[pos++] = s[i]-'A'+'a';
        }
        s.resize(pos);
        for(int i=0;i<s.size()/2;i++) if(s[i]!=s[s.size()-1-i]) return false;
        return true;
    }
};